OrbitCore/ElfFile.cpp

```diff
@@ -0,0 +1,95 @@
+#include "ElfFile.h"
+
+#include <vector>
+
+#include "PrintVar.h"
+#include "OrbitFunction.h"
+#include "Path.h"
+#include "absl/strings/str_format.h"
+#include "llvm/Object/ELFObjectFile.h"
+#include "llvm/Object/ObjectFile.h"
+
+std::unique_ptr<ElfFile> ElfFile::Create(const std::string& file_path) {
+  std::unique_ptr<ElfFile> result(new ElfFile(file_path));
+  if (!result->Load()) {
+    return nullptr;
+  }
+
+  return result;
+}
+
+bool ElfFile::Load() {
+  auto object_file_or_error =
```

Comment:
Explicit type would be more readable instead of auto.

Reply:
done